### ISSUE TITLE
nuitka exe build from av library, building success but got No module named 'av.buffer'

### ISSUE THREAD

**Issue by AnhTalent** [PyAV-Org/PyAV#1120]
Hi @jlaine,

I'm writing a code to transfer a file using the aiortc module.
aiortc imports av package for video or audio streaming.
I successfully made the code and in the python virtual environment, the code transfer the datastream without any issue.

But once I built the exe using nuitka, I got the following error from running the exe console:
...
File "av\audio\frame.pyx", line 1, in init av.audio.frame
File "av\frame.pyx", line 1, in init av.frame
ModuleNotFoundError: No module named 'av.buffer'
...

my environment is :
nuitka : 1.5.6
aiortc : 1.5.0
av : 9.2.0

I hope an urgent help for this issue.

Best,

**Comment by CheshireCC** [comment]
> 

I also got a bug "no module named av.audio.codeccontext" , when I use "av.open()" to open a file.
I build my exe program for Windows10 with nuitka, too.
Do you know how to fix those bugs?

**Comment by AnhTalent** [comment]
@CheshireCC Check the Nuitka Issue :
https://github.com/Nuitka/Nuitka/issues/2192#event-9102025851

**Comment by CheshireCC** [comment]
> @CheshireCC Check the Nuitka Issue : [Nuitka/Nuitka#2192 (comment)](https://github.com/Nuitka/Nuitka/issues/2192#event-9102025851)
Thank You very much!
I checked this issue ,it's seemed that thsi bug is planed to be fixed in version 1.6, but my Nuitka is version 1.7.5 now, why this bug come back again?
I had try to use factory version nuitka, but not work, 
It just still "no module named av.logging" "no module named av.xxxxx"
should I use nuitka 1.6?

**Comment by AnhTalent** [comment]
Yes, I think it should be great to use nuitka 1.6 if it is required to build urgently.
But it would be great to open a new issue on Nuitka git to solve the issue in the latest versions as well.

Btw, what is your av version?

**Comment by CheshireCC** [comment]
@AnhTalent 
I tried av==9.2.0 and av == 10.0.0
they all not work

**Comment by albao666** [comment]
> 

this works fro me, thx

**Comment by albao666** [comment]
> @AnhTalent
> 
> Do you Know what? I fixed this bug with add codes to `av.__init__.py` :
> 
> ```
> from <av.subPackageName.moduleName import <which your program lost>
> ```
> 
> for my poject, I do like this :
> 
> ```
> from av.audio.codeccontext import AudioCodecContext
> ```
> 
> then , all error like `no module named av.logging` disappeared. It looks like a bug with `PyAV`......

this works for me, thx!
